packages/plugin-ext/src/hosted/node/plugin-host-rpc.ts

```diff
@@ -76,6 +76,10 @@ export class PluginHostRPC {
         );
     }
 
+    async terminate(): Promise<void> {
```

Comment:
Hello is it needed ?
as plugin-manager is available in the RPC ?

Reply:
yes, but it is created by this class